test/release/examples/primers/domains.chpl

```diff
@@ -94,15 +91,15 @@ writeln(RDtrans_n);
 var RSD1, RSD2 : subdomain(RD);
 
 // A subdomain is initially empty.
-writeln("RSD1:", RSD1);
-writeln("RSD2:", RSD2);
+writeln("RSD1:", RSD1); // RSD1:{1..0, 1..0, 1..0}
+writeln("RSD2:", RSD2); // RSD2:{1..0, 1..0, 1..0}
 
 // We can select parts of the original rectangular domain using slicing.
-RSD1 = RD[..n/2, .., ..]; // This gives half of the domain
-RSD2 = RD[n/2+1.., .., ..]; // And this the other half.
+RSD1 = RD[..n/2, .., ..];   // This gives half of the domain,
+RSD2 = RD[n/2+1.., .., ..]; // and this gives the other half. 
```

Comment:
Please remove the space at the end of the line.

Reply:
Done, thanks.